lib/closex/http_client.ex

```diff
@@ -95,12 +76,47 @@ defmodule Closex.HTTPClient do
 
   # TODO: at some point we'll need to build a generic pagination/fetch more routine when we hit 50+ users
   @doc "List all users in your organization: https://developer.close.io/#users-list-all-the-users-who-are-members-of-the-same-organizations-as-you-are"
-  def get_users(opts \\ []) do
-    request(:get, "/user/", %{}, [{"Content-Type", "application/json"}], opts) |> handle_response
+  def get_users(limit \\100) do
+    find_all("user", "", limit)
+  end
+
+  def find_all_opportunities(term, limit \\ 100) do
+    find_all("opportunity", term, limit)
   end
 
   # Private stuff...
 
+  defp find(resource, search_term, opts \\[]) do
+    opts = merge_search_term_into_opts(search_term, opts)
+
+    # HACK: Pass empty object as body to workaround bug in HTTPoison, see: https://github.com/edgurgel/httpoison/issues/285
```

Comment:
The issue looks to be fixed now! Can we delete this comment or do we need to change some code?